packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/start-job-run.test.ts

```diff
@@ -1043,12 +1028,137 @@ test('Permitted IAM policy permissions for Custom Resource Lambda', () => {
       }],
```

Comment:
I don't find this check to be useful as it's auto-generated by a construct we call, please remove